src/include/cpu/power/istep_13.h

```diff
@@ -95,17 +100,20 @@ extern mcbist_data_t mem_data;
  * These functions should not be used before setting mem_data.speed to a valid
  * non-0 value.
  */
-static inline uint64_t ps_to_nck(uint64_t ps)
+static inline uint64_t tck_in_ps(void)
 {
 	/*
 	 * Speed is in MT/s, we need to divide it by 2 to get MHz.
 	 * tCK(avg) should be rounded down to the next valid speed bin, which
 	 * corresponds to value obtained by using standardized MT/s values.
 	 */
-	uint64_t tck_in_ps = 1000000 / (mem_data.speed / 2);
+	return 1000000 / (mem_data.speed / 2);
```

Comment:
```suggestion
	return PSEC_PER_USEC / (mem_data.speed / 2);
```

We define a value but do not use it? Or is it something different?